torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -510,9 +510,13 @@ def _lower_static_weighted_ref_module(
         setattr(modules[parent_name], module_name, q_module)
 
         # Step 2: Reroute around dq_node, and remove q_node and its args
-        dq_node = ref_node.args[0]
-        assert(isinstance(dq_node, Node))
-        ref_node.replace_input_with(dq_node, dq_node.args[0])
+        for arg in ref_node.args:
```

Comment:
I feel this might be a bit hacky, generally we want to identify a specific pattern and just lower that pattern. this function (_lower_static_weighted_ref_module) is assuming that we have "dq -> ref_fp32_module -> q" pattern I think, could you 
1). add some checks to this function to make sure this is the case, e.g. check len(ref_node.args) == 1 or something
2). add another lowering function for conv -> add that uses this code?

Reply:
Thanks for the suggestions, @jerryzh168. Followed up these 2 steps:
1. In this PR, I only add the the check of `len(ref_node.args) == 1` for this pass ( `_lower_static_weighted_ref_module`)
2. I have added another lowering pass for `conv -> add` named as `_lower_static_weighted_ref_module_with_two_dq_inputs` in this https://github.com/pytorch/pytorch/pull/91153. Although it is working in progress to cover more cases, could you take a look of this lowering pass we talked about here? Does it look good to you?

Reply:
sounds great, could you update the summary for this PR as well